src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ListView/RotatedListView_WithRotatedItems.xaml

```diff
@@ -0,0 +1,521 @@
+﻿<UserControl x:Class="SamplesApp.Windows_UI_Xaml_Controls.ListView.RotatedListView_WithRotatedItems"
+			 xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+			 xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+			 xmlns:local="using:UITests.Shared.Windows_UI_Xaml_Controls.ListView"
+			 xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+			 xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+			 xmlns:controls="using:Uno.UI.Samples.Controls"
+			 xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+			 xmlns:xamarin="http://uno.ui/xamarin"
+			 mc:Ignorable="d xamarin"
+			 d:DesignHeight="300"
+			 d:DesignWidth="400">
+
+	<UserControl.Resources>
+		<Style TargetType="ListViewItem"
```

Comment:
You should try to reduce the template/styles to the minimum required to illustrate your sample.

Reply:
@carldebilly This is a sample to illustrate a practical uses of the rotatedListView, and not a sample to reproduce an issue, should I still strip away its styling?

Reply:
It depends if the content of the template is important to what the sample is trying to illustrate.